ui/kotlinx-coroutines-javafx/build.gradle.kts

```diff
@@ -3,13 +3,20 @@
  */
 
 plugins {
-    id("org.openjfx.javafxplugin")
+    id("org.openjfx.javafxplugin") version "0.0.9"
```

Comment:
Shouldn't this be done by changing `javafx_plugin_version` in `gradle.properties` instead?

Reply:
Also, it seems that changing the version does nothing to fix the issue: the problem is solved by just making the edit to `configuration` below.

I'm making this comment just for documentation purposes.